docs/ignoring-files-folders-code.md.template

```diff
@@ -223,9 +235,27 @@ Semgrep AppSec Platform users can disable rules and rulesets through the Policie
 
 * [Manage findings](/semgrep-code/triage-remediation#manage-findings)
 * [Ignore findings through PR and MR comments](/docs/semgrep-code/triage-remediation#triage-findings-through-pr-and-mr-comments)
-
 ## Troubleshooting
 
+### Tips to prevent unexpected ignore behavior
+
+:::tip
+This section focuses on ignoring as excluding or skipping files, not as a triage action. 
+:::
+
+Because Semgrep ignore logic is configured at the file, repository, and platform level, you may sometimes encounter unexpected behavior.
+
+- It is recommended to only create an custom, user-defined `.semgrepignore` file if you are **overriding** the Semgrep defaults. This means defining all other items to ignore through the global or project path ignores.
+  - This method works well if your organization primarily scans using the `semgrep ci` command.
+- Be aware that creating a user-defined `.semgrepignore` file enables developers to edit it. Don't exclude the `.semgrepignore` file from Git tracking to keep a log of changes to it.
```

Comment:
```suggestion
- Be aware that creating a user-defined `.semgrepignore` file enables developers to edit it.
- Include the `.semgrepignore` file in Git tracking to keep a log of changes and ensure it's applied consistently.
```
Two negatives -> positive, and also they really should do this anyway. 😅 It's necessary for CI or SMS and even on local dev machines, you'd need to be the same for everyone to avoid messing up findings. Those felt like separate enough points to be their own bullets, but I don't feel strongly about that.